Sources/Core/AWSClientRuntime/Tests/AWSClientRuntimeTests/Customizations/BedrockAPIKeyInterceptorTests.swift

```diff
@@ -0,0 +1,133 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+import ClientRuntime
+import SmithyHTTPAPI
+@testable import AWSClientRuntime
+import Smithy
+import SmithyIdentity
```

Comment:
Unit tests below verify that the interceptor makes the expected changes to the request context.  These test cases are based on the spec for the feature.